src/binary/ibc-relayer/setup-prometheus.ts

```diff
@@ -0,0 +1,58 @@
+import http from 'http';
+
+import client from 'prom-client';
+
+import { Logger } from '../create-logger';
+
+let initialized = false;
+
+function getMetrics() {
+  return {
+    pollCounter: new client.Counter({
+      name: 'poll_counter',
+      help: 'Poll counter',
+    }),
```

Comment:
This is only an example of custom metric that should be removed. Let's replace this with some real ones.

Reply:
Very good start! Counter makes sense here for this metric.  👍 

Some notes for a future version:
I have noticed though that the naming does not fit the others, yet:
```
confio_relayer_nodejs_gc_duration_seconds_count{kind="minor"} 80
poll_counter 13
```
With a common prefix we can find all relayer metrics easily. Otherwise some context is missing as a prometheus instance is likely not exclusive to the relayer.

No idea if the relayer is always bidirectional but as this would involve 2 chains the semantics of poll is not fully clear to me. You can use `labels` to mark which chain you polled.